custom/rch/views.py

```diff
@@ -0,0 +1,88 @@
+from django.utils.translation import ugettext_noop
+from django.views.generic import TemplateView
+from django.core import serializers
+import json
+from custom.rch.forms import BeneficiariesFilterForm
+from custom.rch.models import RCHMother, AreaMapping, RCHChild
+
+
+class BeneficariesList(TemplateView):
```

Comment:
Yup! just views to list down Beneficiaries to show what data has been received from API.